stp_zmq/remote.py

```diff
@@ -76,7 +76,7 @@ def connect(self, context, localPubKey, localSecKey, typ=None):
         sock.curve_serverkey = self.publicKey
         sock.identity = localPubKey
         set_keepalive(sock, self.config)
-        set_zmq_internal_queue_length(sock, self.config)
+        set_zmq_internal_queue_size(sock, self.config.ZMQ_NODE_QUEUE_SIZE)
```

Comment:
This is also used in the old client for client-to-client communication, isn't it? Why do we always set it to `MQ_NODE_QUEUE_SIZE` here?

Reply:
Yes, made it optional.